geniza/footnotes/tests/test_footnotes_migrations.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+from django.test import TransactionTestCase
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+import pytest
+
+
+# migration test case adapted from
+# https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
+# and copied from mep-django
+
+
+@pytest.mark.last
```

Comment:
does the decorator do anything here? does this thing actually get run as a (no-op) test?

Reply:
relatedly: if we apply the decorator here do we also need it on tests that subclass this?

Reply:
I remember testing before to see if I could just put it on the base classes and not the subclasses, and that didn't work; it needs to be on each specific test class. It probably doesn't do anything on this one (except possibly serve as a reminder that subclasses should be run last). Should I remove and add a comment?

Reply:
yeah, that sounds right. i think removing and adding a comment would be perfect